src/main/java/uk/gov/hmcts/ccd/domain/service/startevent/AuthorisedStartEventOperation.java

```diff
@@ -104,7 +105,7 @@ private CaseType getCaseType(String caseTypeId) {
 
     private Set<String> getCaseRoles(CaseDetails caseDetails) {
         if (caseDetails == null || caseDetails.getId() == null || Draft.isDraft(caseDetails.getId())) {
-            return Collections.emptySet();
+            return Collections.singleton(GlobalCaseRole.CREATOR.getRole());
```

Comment:
if you do the refactoring suggested in the comment above, we can remove this duplication here and just call:
`getCaseCreationCaseRoles()`